tests/README.md

```diff
@@ -105,29 +105,29 @@ results summary includes the location of the complete logs, which is logged in
 the form /var/tmp/test_results/[ISO 8601 date].  A normal test run launched
```

Comment:
nit: You may want to put single ticks around `/var/tmp/test_results/[ISO 8601 date]` as well.